Yosemite/YosemiteTests/Mocks/Networking/Remote/MockMediaRemote.swift

```diff
@@ -12,11 +12,8 @@ final class MockMediaRemote {
     /// The results to return based on the given site ID in `loadMedia`
     private var loadMediaResultsBySiteID = [Int64: Result<WordPressMedia, Error>]()
 
-    /// The results to return based on the given site ID in `loadMediaLibrary`
-    private var loadMediaLibraryResultsBySiteID = [Int64: Result<[Media], Error>]()
-
     /// The results to return based on the given site ID in `loadMediaLibraryFromWordPressSite`
```

Comment:
Nit: The comment needs to be renamed as well.

Reply:
Done in 79a826e6f99f5bb5a61bdd5ba77a53bd6c7d7ee8.